docs/Troubleshooting.md

```diff
@@ -25,32 +24,31 @@ import 'rxjs';
 ```
 This will add every core RxJS operator to the `Observable` prototype.
 
-#### Add only the operators you use
+### Add only the operators you use
 
-tl;dr
+There are several ways to do this. Check out the [RxJS documentation on this](https://github.com/ReactiveX/rxjs#installation-and-usage).
 
+For example,
 
 ```js
 import 'rxjs/add/operator/switchMap';
 action$.ofType(...).switchMap(...);
 
 // OR
 
-import { switchMap } from 'rxjs/operator/switchMap';
-action$.ofType(...)::switchMap(...);
+import { switchMap } from 'rxjs/operators';
+action$.ofType(...).switchMap(...);
```

Comment:
This was actually not a typo. It was just an example using the [bind operator](https://github.com/tc39/proposal-bind-operator) proposal. But that proposal is unlikely to ever make it, so removing that example is probably a good idea to remove it.

```suggestion
import { switchMap } from 'rxjs/operators';
import { ofType } from 'redux-observable';
// If you are using a compiler with pipeline operator support (non-standard)
action$ |> ofType(...) |> switchMap(...);
```

Reply:
Alternatively we could remove this suggestion altogether, since the pipeline operator is also non-standard still. Though there is more hope it might one day land.